### PR TITLE
handle exceptions for logger processing

### PR DESCRIPTION
**Pull request by aniketmaurya**
<details>
  <summary><b>Before submitting</b></summary>

- [ ] Was this discussed/agreed via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/pytorch-lightning/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

</details>

## ⚠️ How does this PR impact the user? ⚠️

Avoid stopping logger process dues to exception. 

---- 

## What does this PR do?

Avoid stopping logger process dues to exception. 

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in GitHub issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: src/litserve/loggers.py [hunk 1]
```diff
@@ -112,7 +112,10 @@ def _process_logger_queue(loggers: List[Logger], queue):
         while True:
             key, value = queue.get()
             for logger in loggers:
-                logger.process(key, value)
+                try:
+                    logger.process(key, value)
+                except Exception as e:
+                    module_logger.error(f"Error processing log entry with key {key} and value {value}: {e}")
```
Comment (Andrei-Aksionov) [inline src/litserve/loggers.py:118]: Since there might be multiple loggers, it would be useful to also log exactly what logger failed to log.

Comment (lantiga) [reply src/litserve/loggers.py:118]: I agree

Comment (aniketmaurya) [reply src/litserve/loggers.py:118]: thanks for feedback @Andrei-Aksionov @lantiga! Would logging the exception `logger.error(f"... {e}")` show the error from exactly which logger it failed?

Comment (aniketmaurya) [reply src/litserve/loggers.py]: Added `logger.__class__.__name__` in the error msg

**Review by Andrei-Aksionov — APPROVED** [review]

**Pull request merged**